python/hail/matrixtable.py

```diff
@@ -2031,6 +2031,15 @@ def cols(self) -> Table:
         """
         return Table(self._jvds.colsTable())
 
+    def col_keys(self) -> Table:
```

Comment:
@tpoterba I added this when PCRelate was being developed, I don't totally remember why, but I remember you and I disagreeing about whether it should exist.

Reply:
I think it was to be able to collect the field easily, yeah? I think that's totally covered with the new expression methods

Reply:
So now this should just be `ds.col_key.collect()`?

Reply:
yes, that'll work now :)